R/build_lm.R

```diff
@@ -1,3 +1,52 @@
+
+
+# Builds partial_dependency object for lm/glm with same structure (a data.frame with attributes.) as edarf::partial_dependence.
+partial_dependence.lm_exploratory = function(fit, target, vars = colnames(data),
+  n = c(min(nrow(unique(data[, vars, drop = FALSE])), 25L), nrow(data)),
+  interaction = FALSE, uniform = TRUE, data, ...) {
+
+  predict.fun = function(object, newdata) {
+    predict(object, newdata = newdata, type = "response")
+  }
+
+  aggregate.fun = function(x) {
+    c("preds" = mean(x))
+  }
+
+  args = list(
+    "data" = data,
+    "vars" = vars,
+    "n" = n,
+    "model" = fit,
+    "uniform" = uniform,
+    "predict.fun" = predict.fun,
+    "aggregate.fun" = aggregate.fun,
+    ...
+  )
+  
+  if (length(vars) > 1L & !interaction) {
```

Comment:
Please add some comment for this condition.

Reply:
Added comment.